dcpy/test/connectors/edm/test_packaging.py

```diff
@@ -0,0 +1,67 @@
+import pytest
+import os
+from dcpy.connectors.edm import publishing, packaging
+
+
+def test_bucket_empty(create_buckets, mock_data_constants):
```

Comment:
Generally we want to avoid tests depending on each other, or depending on order of execution. Maybe we move this check into the other tests?